### PR TITLE
Add X Air Edit from Behringer for XR mixers

### PR DESCRIPTION
**Pull request by valentindavid**

### CODE REVIEW

File: com.behringer.XAirEdit.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - install -Dm644 -t /app/share/icons/hicolor/512x512/apps com.behringer.XAirEdit.png
+  sources:
+  - type: script
+    dest-filename: apply_extra
+    commands:
+    - tar xvf X-AIR-Edit_LINUX_X64_V1.5.tar.gz
+  - type: file
+    path: com.behringer.XAirEdit.desktop
+  - type: file
+    path: com.behringer.XAirEdit.appdata.xml
```
Comment (LeandroStanger) [inline com.behringer.XAirEdit.yml:31]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

**Comment by LeandroStanger** [comment]
bot, build com.behringer.XAirEdit

**Comment by valentindavid** [comment]
What do do about screenshots? Is there a place I can upload some?

**Comment by valentindavid** [comment]
bot, build com.behringer.XAirEdit

**Comment by valentindavid** [comment]
bot, build com.behringer.XAirEdit

**Comment by valentindavid** [comment]
I have put a temporary url to the screenshot, it has to be fixed when the repository is created.

File: com.behringer.XAirEdit.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+app-id: com.behringer.XAirEdit
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: X-AIR-Edit
+tags:
+- proprietary
+finish-args:
```
Comment (barthalion) [inline com.behringer.XAirEdit.yml:8]: How does that work, `--device=all` is not needed to connect(?) to the external device?

Comment (barthalion) [reply com.behringer.XAirEdit.yml:8]: Ah, it's fully digital. Never mind!

**Pull request closed without merging**